chain/bitcoind_client.go

```diff
@@ -981,17 +1008,21 @@ func (c *BitcoindClient) FilterBlocks(
 
 	blockFilterer := NewBlockFilterer(c.chainConn.cfg.ChainParams, req)
 
+	storeHash := make([]*chainhash.Hash, 0, len(req.Blocks))
+
+	for _, block := range req.Blocks {
```

Comment:
nit: better to use the form: `for i := range req.Blocks` and then refer to the i'th block as `req.Blocks[i]`.